composer.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "juststeveking/laravel-envoyer-sdk",
+  "name": "node-dog-consulting/laravel-envoyer-sdk",
```

Comment:
You probably renamed it for you own use, this is why PRs should come from there own branch and not `main`.